static_frame/core/store.py

```diff
@@ -273,6 +190,121 @@ def __hash__(self) -> int:
         return self._hash
 
 
+class StoreConfig(StoreConfigHE):
+    '''
+    A read-only container of parameters used by :obj:`Store` subclasses for reading from and writing to multi-table storage formats.
+    '''
+    label_encoder: tp.Optional[tp.Callable[[tp.Hashable], str]]
+    label_decoder: tp.Optional[tp.Callable[[str], tp.Hashable]]
+
+    __slots__ = (
+            'label_encoder',
+            'label_decoder',
+            )
```

Comment:
Is this implicitly a union of the parent class? I generally re-define all the slots needed in derived class, but if this is the union, it is an efficient representation.

Reply:
Yeah - I want all my parent's slots + these two new ones.

Reply:
Maybe except `_hash`, although that won't matter since I explicitly raise on `__hash__()` ...